### PR TITLE
People subpage

### PR DESCRIPTION
**Pull request by jakubkedzierski92**
Added scrolling to description components, everygthing looks fine, need to add some additional props. Pagination might cause problem cause i changed to mine.

### CODE REVIEW

File: src/features/movieList/Movie/index.js
Comment (Kubabogd) [inline src/features/movieList/Movie/index.js:28]: I'm not sure but I think we'll need it. You probably got an error the first time the page was loaded and you had to comment on it? I did it this way because we only need to display the year instead of the exact date. Let me know if this comment is needed.

File: src/queryParameters 2.js
Comment (Werka481) [inline src/queryParameters 2.js:1]: This file is already on the master branch. I think it should be removed before the merge.

File: src/searchQueryParamName 2.js
Comment (Werka481) [inline src/searchQueryParamName 2.js:1]: This file is already on the master branch. I think it should be removed before the merge.

**Review by Werka481 — APPROVED** [review]
I think we shouldn't merge this pagination just yet, because it's infinite. The rest of the code seems fine.

**Pull request merged**
